Kickstarter-iOS/Views/Cells/CommentViewMoreRepliesFailedCell.swift

```diff
@@ -0,0 +1,102 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class CommentViewMoreRepliesFailedCell: UITableViewCell, ValueCell {
```

Comment:
There's no view model here because the cell is dumb and only needs to be rendered/tapped.